packages/test/test-end-to-end-tests/src/test/benchmark/LoadDocument.time.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { ITestObjectProvider } from "@fluidframework/test-utils";
+import { describeE2EDocsRuntime } from "@fluidframework/test-version-utils";
+import { benchmark } from "@fluid-tools/benchmark";
+import { DocumentCreator } from "./DocumentCreator";
+import { DocumentMap } from "./DocumentMap";
+
+const scenarioTitle = "Load Document";
+describeE2EDocsRuntime(scenarioTitle, (getTestObjectProvider, getDocumentInfo) => {
+	let documentMap: DocumentMap;
+	let provider: ITestObjectProvider;
+
+	before(async () => {
+		provider = getTestObjectProvider();
+		const docData = getDocumentInfo(); // returns the document to be processed.
```

Comment:
```suggestion
		const docData = getDocumentInfo(); // returns the type of document to be processed.
```
I think it's an important distinction here, since this doesn't give you the actual document.